_life/money.md

```diff
@@ -79,75 +71,74 @@ Do not expect good yields from non-investment bank accounts, even the savings on
 Make sure you compare fees before choosing a solution: keeping it in cash should be free everywhere, but if you want to invest, your bank’s offering probably won’t be the cheapest.
+At withdrawal, a progressive tax rate is applied depending on the capital amount and your canton of residence. ([source](https://finpension.ch/en/capital-withdrawal-tax-compared/))
 
-In general, it’s not worth it to subscribe to a joint life insurance / third pillar product (a.k.a. "Pillar 3b"), even though salespeople may try to convince you otherwise.
+There are retirement accounts called "Pillar 3b" which are not tax deductible. Salespeople will try to convince you these are useful, but they are most likely wrong.
 
 
 ## Taxes
 
-Switzerland has taxes on your income (i.e., salary) and your wealth (i.e., money in your bank account(s), possessions such as a car).
-Taxes are levied by the federal government, cantons, and communes; this means the amount you pay in tax can vary depending on where you live in the country.
-
-There are two ways to pay taxes: "at the source", default for people who are not permanent residents, and with "acomptes", for Swiss citizens, permanent residents, and some other cases.
-Paying taxes at the source means they are deducted directly from your salary, with no action necessary on your part.
-Paying taxes with acomptes means you get a bill at the start of the year and have to pay part of it each month, with the easiest option being to register for an e-Bill on your online banking so approval is one click once a year.
+The general principles of Swiss taxation are:
+- Your income and your wealth are subject to taxation, no matter where you earned or store them in the world.
+- Taxes are levied by the federal government, by cantons, and by communes, but the tax declaration is a single online form.
+- If you are married, you must file jointly.
+- Permanent residents must pay an advance on their taxes each month, while non-permanent residents have their taxes deducted directly from their salary.
+- The tax authorities will help you as long as you are honest, never lie nor try to hide money, and do not try to find loopholes or technicalities.
+- If you realize you filed something incorrectly, or forgot to declare something, it's always better to contact the tax authorities yourself than wait for them to find out.
 
-If you have a B permit, _AND_ your net worth is lower than 50,000 CHF, _AND_ your yearly income is lower than 120,000 CHF, _AND_ you have no investments such as indexed funds,
-_AND_ you do not have a 3a pillar or other significant deductions,
-you do not need to do anything for taxes. You are taxed at the source and that's it.
+> ⚠️ The remainder of this page is a helpful guide to common taxable items and deductions _in the canton of Vaud_, not tax advice about your personal situation.
+> You should still read every field in the tax form at least once to ensure you do not forget anything.
```

Comment:
Hmm, that's a blockquote, which might serve the visual purpose but is the wrong semantic element. I think putting the whole paragraph in italics might be better. Plus maybe another warning emoji at the end too.